.github/workflows/push-treasurenet-mainnet-artifacts-to-s3.yml

```
@@ -0,0 +1,178 @@
+          for node_type in ${{ env.NODE_TYPES }}; do
+            max=${{ env.GENESIS_VALIDATOR_COUNT }}
+            if [ "$node_type" = "rpc" ]; then max=${{ env.RPC_NODE_COUNT }}; fi
+            if [ "$node_type" = "bootnode" ]; then max=${{ env.BOOTNODE_COUNT }}; fi
+            
+            for i in $(seq 1 $max); do
+              tar -zcvf ./${node_type}-${i}.tar.gz -C ${{ env.DATA_PATH }}/${node_type}-${i} .
+            done
+          done
+
+      - name: Upload artifacts to AWS S3
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:12:30: Double quote to prevent globbing and word splitting

(shellcheck)